srcpkgs/thunderbird/template

```diff
@@ -35,6 +35,10 @@ case $XBPS_TARGET_MACHINE in
 	ppc*) broken="xptcall bitrot" ;;
 esac
 
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" rust-std"
```

Comment:
why is this needed?

Reply:
> why is this needed?

Not quite sure, but it doesn't compile on aarch64/armv7l without it. It complains about not finding rustc_driver IIRC.

Reply:
hm, I just tested aarch64 without that and it worked fine. testing armv7l-musl now

Reply:
armv7l-musl was also fine without it